python_modules/dagster/dagster_tests/core_tests/snap_tests/test_repository_snap.py

```diff
@@ -345,3 +346,82 @@ def my_asset(foo: MyStringResource):
     assert {consumer.name for consumer in env_vars["MY_CONFIG_NESTED_STRING"]} == {"quuz"}
     assert "MY_CONFIG_LIST_NESTED_STRING" in env_vars
     assert {consumer.name for consumer in env_vars["MY_CONFIG_LIST_NESTED_STRING"]} == {"quuz"}
+
+
+def test_repository_snap_definitions_resources_assets_usage() -> None:
+    class MyResource(ConfigurableResource):
```

Comment:
maybe an included resource with no usage just to test the nil case

Reply:
good idea